host/err.go

```diff
@@ -0,0 +1,18 @@
+package host
+
+import "errors"
+
+var (
+	// ErrInvalidEntryName is used on RegisterEntry to
+	// enforce the use of known entry points.
+	ErrInvalidEntryName = errors.New("unknown entry")
```

Comment:
Either update error name to `ErrUnknownEntry` or the error message to "invalid entry name" to be consistent.